src/UserActivity.js

```diff
@@ -61,32 +61,33 @@ export default class UserActivity {
     }
 
     /**
-     * Runs the given timer while the user is 'active', aborting when the user becomes 'passive'.
-     * See userCurrentlyActive() for what it means for a user to be 'active'.
+     * Runs the given timer while the user is 'active', aborting when the user is no longer
+     * considered currently active.
+     * See userActiveNow() for what it means for a user to be 'active'.
      * Can be called multiple times with the same already running timer, which is a NO-OP.
      * Can be called before the user becomes active, in which case it is only started
      * later on when the user does become active.
      * @param {Timer} timer the timer to use
      */
     timeWhileActive(timer) {
```

Comment:
Hmm, shouldn't we carry the new naming through to this API as well? We'd then have `timeWhileActiveNow` and `timeWhileActiveRecently`.